app/controllers/api/encryptables_controller.rb

```diff
@@ -72,6 +72,18 @@ def model_class
   end
   # rubocop:enable Metrics/MethodLength
 
+  def fetch_entries
+    if encryptable_file?
+      super
+    else
+      encryptables = Encryptables::FilteredList.new(current_user, params).fetch_entries
```

Comment:
This line might be completely useless if `tag_param.present?` is true as then it will just overwrite encryptables :grin: 
Make it so that just one of the two lines is run depending on the condition.
This will also allow you to use the ruby functionality of returning the last argument of a method instead of having to specifically add a line just for `encryptables` again.